libp2p/network/utils_test.go

```diff
@@ -0,0 +1,183 @@
+package network
+
+import (
+	"fmt"
+	"sort"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/golang/mock/gomock"
+	"github.com/pokt-network/pocket/p2p/providers/addrbook_provider"
+	"github.com/pokt-network/pocket/p2p/providers/current_height_provider"
+	typesP2P "github.com/pokt-network/pocket/p2p/types"
+	mock_typesP2P "github.com/pokt-network/pocket/p2p/types/mocks"
+	"github.com/pokt-network/pocket/runtime"
+	"github.com/pokt-network/pocket/runtime/configs"
+	configTypes "github.com/pokt-network/pocket/runtime/configs/types"
+	"github.com/pokt-network/pocket/runtime/genesis"
+	coreTypes "github.com/pokt-network/pocket/shared/core/types"
+	cryptoPocket "github.com/pokt-network/pocket/shared/crypto"
+	"github.com/pokt-network/pocket/shared/modules"
+	"github.com/pokt-network/pocket/shared/modules/mocks"
+)
+
+const (
+	maxNumKeys             = 42
+	genesisConfigSeedStart = 42
+)
+
+var (
+	keys                 []cryptoPocket.PrivateKey
+	serviceUrlFormat     = "node%d.consensus:42069"
```

Comment:
Thinking out loud

I wonder if this could cause issues with the fact that currently we have two different naming conventions for nodes that are in `docker compose` and nodes in `kubernetes`...

For the K8s ones we have the naming convention **v1-validatorXXX**

I'll keep reading the code and perhaps add more color as I am learning it.

Reply:
Good looking out!

This is just used in these unit tests so none of the localnet infrastructure is in play here.

Reply:
Oh I removed that comment after I realised the same thing but I guess VSCode didn't send the memo to github 🤣